docs/preferences.md

```diff
@@ -0,0 +1,147 @@
+# Preferences
+
+## General
+
+The **General** tab, provides general information about the project's status as well as links to discuss the project, report issues, or to learn more about the project.
+
+### Check for updates automatically
+
+When an update is available, users will be provided a notification and the release notes for the upgrade target. This happens whether automatic updates are enabled or not. If this option is enabled, the update will be downloaded and then installed the next time Rancher Desktop is started.
+
+### Allow collection of anonymous statistics to help us improve Rancher Desktop
+
+This option allows Rancher Desktop to collect information on how you interact with the Rancher Desktop application. Information such as what workloads you run are not collected.
+
+## Kubernetes Settings
+
+On the **Kubernetes Settings** tab, you can manage the settings of your virtual machine.
+
+### Kubernetes Version
+
+This option presents a list of Kubernetes versions that your Rancher Desktop instance can use. 
+
+When upgrading:
+
+- A check will is performed to see if the target Kubernetes version is available locally. If not, it will download the files.
```

Comment:
A check `is` performed or a check `will be` performed, this looks like an incomplete edit.